### ISSUE TITLE
Chaff combi mode driver head on collision with chopper

### ISSUE THREAD

**Issue by JDTiger** [Courseplay/courseplay#1578]
Hey, a problem i already had sometimes in FS15, but always in FS17 is a headon collision betwenn a Chopper on Fieldwork mode and the second transporter on combi mode.
So the chopper is harvesting Corn, thios transporter, wich is currently beeing filled reaches 50% and the second tractor drives straight into the chopperhead...
![fsscreen_2017_01_01_23_31_16](https://cloud.githubusercontent.com/assets/9157835/21583301/9738bfd2-d07b-11e6-9ad2-562f630312dc.png)

**Comment by pops64** [comment]
Can confirm this is worse than in FS15. I think there is way  point that is generated but it is getting skipped over.  Cause if I remember correctly it is suppose to drive to some point away and behind the tractor before turning around. I remember seeing this point generated on the CP hud but went away before I could read what it was. Another bug related to this function, the speed over the 2nd follower(The one waiting for the current loading tipper to drive off) is not receiving a speed. I have noticed this tractor is set to 26 mph and not the 6 or 8 mph that it is suppose to be. I assuming this a problem on the whole because even the combines are having issues sending the right one to its current tipper(speed will jump between 4-10 mph). Sorry for writing a book. As always thanks and keep up the great work

P.S short term fix for this would be jacking up the start at % to 90 to 100

**Comment by ThomasGaertner** [comment]
I will check it when I fix #1527 ...

**Comment by ThomasGaertner** [comment]
please test again with 01.0012 or higher and report ....

**Comment by Raints** [comment]
Not sure where things go wrong, but now with version 016 every second driver in mode2 hit to chopper. It is weird since I never had any issues before with previous versions.

**Comment by ThomasGaertner** [comment]
can you give me a screenshot ? and a more detailed description when it happens

**Comment by Raints** [comment]
@ThomasGaertner  Sry about that. Somehow cant get that anymore. Will test more.

**Comment by ThomasGaertner** [comment]
Ok , next attempt is v 0038: try and report, if it occours again , please add a screenshot to you feedback thanks

**Comment by nyfarmboy07** [comment]
GCV - Field 5 - cp 5.01.00040
NH 850 forager with corn header - T6 & Agroliner + CIH 1455XL & Agroliner 

I have had better luck with the second driver not running into the forager, but it still will happen if the second driver pulls up behind the forager and the forager reaches the end of the row.  

A new issue is the forager will turn around and the driver will not always pull far enough out of the way for the forager to finish the turn around maneuver and line up on the next pass.  

Also, the drivers usually try to line up on the crop side of the forager instead of the harvested side.  
At the end of the row they pull in front of the forager instead of turning away, which is part of the reason the forager will not be able to fully turn around.  I will be adding screen shots later today hopefully.

**Comment by nyfarmboy07** [comment]
I did more testing and found that the forager works correctly when CP is harvesting the field.  I must have hit the hire worker key, as when the forager is running only under the hire worker is the unloading courses backward. I still have issues with the chopper getting run into from time to time.

![fsscreen_2017_01_22_14_07_37](https://cloud.githubusercontent.com/assets/24856526/22207446/308b2e36-e14d-11e6-92db-45d3adc73971.png)

![fsscreen_2017_01_22_13_04_32](https://cloud.githubusercontent.com/assets/24856526/22207517/76a35768-e14d-11e6-9a2b-6d074a355342.png)

![fsscreen_2017_01_22_14_08_56](https://cloud.githubusercontent.com/assets/24856526/22207555/9bf74718-e14d-11e6-861b-0bd9ad50b02a.png)

**Comment by ThomasGaertner** [comment]
please try again with v.0055 or higher and report

**Comment by nyfarmboy07** [comment]
I'm on 000.56, and I found that with CP harvesting or Giants hired worker the results appear to be about the same.  I have found that both conditions allow the tipper operators to select the proper side of the forager.  The major difference is the CP worker will start into the row even if the trailer is out of the range for the spout, the Giants worker will not do this.  This causes the truck and a longer trailer to move into the path of the forager. The shorter trailers seem to adapt a bit better, but as it appears to me, they attempt to make an excessively sharp turn from behind the forager to the loading point.  I attempted changing the turn radius setting without any real results.
  I apologize for the delay, but I have a lot of silage to sell, which CP appears to have an issue there also.   I realize this is not the place, but when unloading the silo, the loader works flawless until (for a reason I do not understand) the loader will enter the silo adjacent to the silo the course is recorded on.  I stop the driver and move the loader back to the start point and he continues to unload until he randomly enters the adjacent silo again. 
  These pictures hopefully will show the issue when the truck attempts to move from the waiting point to the forager.  The way my course is set up, the CP tippers can run into each other while harvesting the headlands, but typically, they will not cross paths after the initial 3 headland passes. 
![fsscreen_2017_01_30_09_35_07](https://cloud.githubusercontent.com/assets/24856526/22573707/3b41f46c-e978-11e6-857b-e6b90ccc7dd8.png)
![fsscreen_2017_01_29_16_02_53](https://cloud.githubusercontent.com/assets/24856526/22573709/3dbae050-e978-11e6-83d6-c0bd897159ab.png)
![fsscreen_2017_01_29_15_43_53](https://cloud.githubusercontent.com/assets/24856526/22573720/4797f356-e978-11e6-940d-461f9274c300.png)

**Comment by ThomasGaertner** [comment]
which side offset is set ?

**Comment by nyfarmboy07** [comment]
I tried adjusting the side offsets from "auto".  If I decreased the offset to move the trailers closer, the truck would stop and the message for "vehicle blocked" would appear.  If I increased the offset, then the situation would happen nearly every time.  

I think the point where the trailer decides which side to drive on needs to be moved back to allow the trailers more room to turn into place.
